### DISCUSSION TITLE
What is the aws hosting cost when you use the terraform script ?

### DISCUSSION THREAD

**Discussion by mozexty** [serverpod/serverpod#322] [Q&A] [answered]
Do you have an idea of what it costs on AWS if you use the terraform script, with no free-tier ?

**Comment by vlidholt** [comment, marked as answer]
I haven't done any detailed calculations, so this is not financial advice. My guess is less than $40/mo, but it may depend on the amount of traffic too. The main cost is the load balancer, database, and the EC2 instance. If you are running a very small project, you can cut out the load balancer or use a smaller database and save some on that too.

Load balancer: ~$16 + traffic
EC2 instance: ~$8
RDS database: ~$13

All other stuff is on-demand pricing (e.g. S3 storage, dns, etc)
